README.md

```diff
@@ -28,21 +28,19 @@ You can however set it to `rmw_fastrtps_dynamic_cpp` using the environment varia
 
 ## Advance usage
 
-`rmw_fastrtps` sets some of the Fast-RTPS configurable parameters:
-* History memory policy: `PREALLOCATED_WITH_REALLOC_MEMORY_MODE`
-* Publication mode: `ASYNCHRONOUS_PUBLISH_MODE`
```

Comment:
Before, asynchronous publish mode was forced, and the readme documented how to configure fastrtps in synchronous publish mode.
Does make sense now to include in the readme how to configure fastrtps in asynchronous mode?
Just wondering, I don't know in which cases the user would be interested in doing that.

Reply:
We are only documenting how to avoid the parameters that are overridden by the code in this repo. Configuring fastrtps for asynchronous mode will now be done as with the rest of the parameters, i.e. with the XML file.

Now that I read it in detail, I think the whole section may need some rework, but the changes will not be related to this PR, so I propose to make them on a separate one.

Reply:
Sounds reasonable